components/netlify/sources/common/common.mjs

```diff
@@ -0,0 +1,81 @@
+import netlify from "../../netlify.app.mjs";
+
+export default {
```

Comment:
I think the convention in this repo is to omit "common" from a [common file](https://pipedream.com/docs/components/guidelines/#common-files-optional) when it's in the `common/` folder. I'd suggest renaming this file to `webhook.mjs` (e.g., Eventbrite [`webhook.mjs`](https://github.com/PipedreamHQ/pipedream/blob/55236b3aa993cbcb545e245803d8654c6358b0a2/components/eventbrite/sources/common/webhook.js)).

Reply:
Addressed